en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
 name: Ranking Criteria
 ---
 
-
 # Quaver Ranking Criteria
 
-In order to get a mapset ranked, it must follow the criteria listed below. In the event that it does not, you may be asked to modify your mapset and resubmit it again for rank.
+In order to get a mapset ranked, it must follow the ranking criteria listed below. In the event of it breaking the criteria you may be asked to modify your mapset.
+
+The ranking criteria is divided into rules and guidelines.
+* Rules are, literally, rules that may not be broken under any circumstances.
+* Guidelines try to guide the mapper and may be ignored when deemed reasonable.
+
+If you're looking for the steps of getting a mapset ranked, see: [Quaver Ranking Process](/docs/Ranking/Process).
+
+# Metadata
+
+#### Rules
+
+* **Metadata must be consistent across difficulties.** The only exception being specific map-descriptive tags, such as pattern names, that improve the searchability of singular difficulties.
+
+## Artist
+
+#### Rules
+
+* **The artist must be absolutely correct.**
+* **The artist must be romanized.** Non-romanized artist can be added to tags.
+* **The artist must not repeat in tags.** Only exception being the case where the artist appears in a relevant tag, such as song genre.
+* If there is no existing person to credit as the artist, `Unknown Artist` must be used.
+
+## Title
+
+#### Rules
+
+* **The title must be absolutely correct.**
+* **The title must be romanized.** Non-romanized title can be added to tags. Loan words from other languages should use the original words instead of attempted romanization.
```

Comment:
it should also be known that the normal Latin script should be used for this, not Latin Extended to avoid cases like this https://quavergame.com/mapset/map/70637

Reply:
If you want to do this, "anglicized" (maybe there's a more appropriate term?) should be added to romanized.

There are better ways to work around this though, like having both an original and romanised title/artist field in the map metadata, and a way to toggle between which one shows up in song selection.